templates/web-application/swc.build.js

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+
+// Can play with most of the important configuration option with the online playground: https://swc.rs/playground
+
+/** @type {import("@swc/core").Config} */
+export default {
+    jsc: {
+        parser: {
+            syntax: "typescript",
+            tsx: true
+        },
+        // The output environment that the code will be compiled for.
+        target: "es2022",
+        transform: {
```

Comment:
VS https://github.com/patricklafrance/pnpm-ts5-swc-esm-jest/blob/master/swc.build.js, I remove the `minify` section.

TL:DR: We don't need to optimise as soon in the packaging. It's have been move to the `optimization` step of Webpack.

Details from SWC: 
- https://swc.rs/docs/configuration/minification
   ![image](https://user-images.githubusercontent.com/4323800/236929708-cc34e202-b385-4b43-99ee-4e168573467e.png)
- https://webpack.js.org/plugins/terser-webpack-plugin/#swc

Reply:
Good catch!